### PR TITLE
Feature/1770 Set page title for pages using either navigation404 or custom 404.

### PR DESCRIPTION
**Pull request by owenmorrill**
#1770

### CODE REVIEW

File: modules/custom/cu_core/cu_core.module [hunk 1]
```diff
@@ -170,6 +170,14 @@ function cu_core_page_alter(&$page) {
       unset($page['help']['system_help']);
     }
   }
+
+  // Fix 404 google analytics tracking
+  global $base_path;
```
Comment (jeor0980) [inline modules/custom/cu_core/cu_core.module]: is this global variable used?

File: modules/custom/cu_core/cu_core.module [hunk 2]
```diff
@@ -170,6 +170,14 @@ function cu_core_page_alter(&$page) {
...
+  $custom_404_page = variable_get('site_404', '');
+  $current_path = arg(0) . '/' . arg(1);
+  if ($custom_404_page === $current_path || arg(0) === 'navigation404') {
```
Comment (jeor0980) [inline modules/custom/cu_core/cu_core.module]: what are `arg(0)` and `arg(1)`?

Comment (alexfinnarn) [reply modules/custom/cu_core/cu_core.module]: Those are "magical" Drupal variables taken from the URI, e.g. "node/1/view" arg(0) = "node, arg(1) = 1, arg(2) = view. Not a great way to get those values but it is a Drupal thing.

**Comment by alexfinnarn** [comment]
closing this in favor of https://github.com/CuBoulder/express/pull/2289/files for now.

**Pull request closed without merging**